src/components/touchableOpacity/index.js

```diff
@@ -117,7 +119,28 @@ export default class TouchableOpacity extends PureBaseComponent {
     );
   }
 
+  render() {
+    const {useSafeArea, safeAreaBackgroundColor} = this.getThemeProps();
+
+    if (useSafeArea && Constants.isIphoneX) {
```

Comment:
try instead rendering a `SafeAreaView` inside as a child. all you need here is to add the extra padding

Reply:
Done